agent/functional_tests/util/utils_windows.go

```diff
@@ -215,564 +115,24 @@ func (agent *TestAgent) StopAgent() error {
 	return agent.Process.Kill()
 }
 
-func (agent *TestAgent) StartAgent()  error {
-  agentInvoke := exec.Command("C:\\GoWrk\\src\\github.com\\aws\\amazon-ecs-agent\\agent")
-  agentInvoke.Start()
-  agent.IntrospectionURL = "http://localhost:51678"
-
-	// Wait up to 10s for it to register
-	var localMetadata handlers.MetadataResponse
-	for i := 0; i < 10; i++ {
-		func() {
-			agentMetadataResp, err := http.Get(agent.IntrospectionURL + "/v1/metadata")
-			if err != nil {
-				return
-			}
-			metadata, err := ioutil.ReadAll(agentMetadataResp.Body)
-			if err != nil {
-				return
-			}
-
-			json.Unmarshal(metadata, &localMetadata)
-		}()
-		if localMetadata.ContainerInstanceArn != nil && *localMetadata.ContainerInstanceArn != "" {
-			break
-		}
-		time.Sleep(1 * time.Second)
-	}
-
-	if localMetadata.ContainerInstanceArn == nil {
-		agent.DockerClient.StopContainer(agent.DockerID, 1)
-		return errors.New("Could not get agent metadata after launching it")
-	}
-
-	agent.ContainerInstanceArn = *localMetadata.ContainerInstanceArn
-	fmt.Println("Container InstanceArn:", agent.ContainerInstanceArn)
-	agent.Cluster = localMetadata.Cluster
-	if localMetadata.Version != "" {
-		versionNumberRegex := regexp.MustCompile(` v(\d+\.\d+\.\d+) `)
-		versionNumberStr := versionNumberRegex.FindStringSubmatch(localMetadata.Version)
-		if len(versionNumberStr) == 2 {
-			agent.Version = string(versionNumberStr[1])
-		}
+func (agent *TestAgent) StartAgent() error {
+	agentInvoke := exec.Command(".\\agent.exe")
```

Comment:
[This README](https://github.com/samuelkarp/amazon-ecs-agent/blob/8d78cecbeeebcdb1ab818f89cfe48ae76dbf5164/agent/functional_tests/README.md#windows-setup) covers Windows functional tests.  We do need changes to the general README for Windows though.